lib/sorts/quick-sort/index.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Sorting function should take 2 arguments
- array of elements
- function comparator, to compare elements